platform/src/main/java/org/stellar/anchor/platform/config/PropertyEventConfig.java

```diff
@@ -31,15 +26,49 @@ public void validate(Object target, Errors errors) {
       return;
     }
 
-    ValidationUtils.rejectIfEmptyOrWhitespace(errors, "publisherType", "");
+    // Validate publisher type
+    ValidationUtils.rejectIfEmptyOrWhitespace(errors, "publisher.type", "");
+
+    PublisherConfig publisherConfig = config.getPublisher();
+    String publisherType = publisherConfig.getType();
+    switch (publisherType) {
+      case "msk":
```

Comment:
I also agree it is less confusing if we have the same order. However, this is separation between the yaml file and the implementation. For YAML, I wish to put `kafka` as the first option. But the implementation, I tried to avoid duplicated calls by a `breakless` `case` statement. That's why MSK goes before KAFKA.